paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -665,3 +672,13 @@ class Paparazzi @JvmOverloads constructor(
       }
   }
 }
+
+fun DeviceConfig.maybeForAccessibility(accessibilityRenderExtensionEnabled: Boolean): DeviceConfig =
```

Comment:
move extension function into Paparazzi class and inline the `accessibilityRenderExtensionEnabled` logic?